src/main/java/cinema/model/MovieSession.java

```diff
@@ -0,0 +1,63 @@
+package cinema.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(cascade = CascadeType.ALL)
```

Comment:
this is for your info, but keep in mind that toOne relations have fetchType eager by default, which is not always quite good